client/components/common/ReactDayPicker/Styles.jsx

```diff
@@ -15,6 +15,7 @@ const Styles = ({ range }) => {
       background: ${theme.palette.primary.dark};
       border-radius: 5px;
       min-width: 268px;
+      z-index: 5000;
```

Comment:
After adding `zIndex: 1` to DatePicker.jsx selectorPopUp, is z-index still needed here?

Reply:
+1

I would also recommend using a value that is more consistent with the others, as opposed to an arbitrary large number like `5000` as it will make it easier to manage all the other z-indexes in the future.

Reply:
Just updated Styles.jsx.